akka-actor/src/main/scala/akka/pattern/BackoffOptions.scala

```diff
@@ -318,6 +330,7 @@ private final case class BackoffOptionsImpl(
   minBackoff:         FiniteDuration,
```

Comment:
Mima reports binary compatibility issue with BackoffOptionsImpl class.
Since the class is private and used only in BackoffOptions, I'm concluding that changing it shouldn't raise a binary compatibility issue and it can be ignored. am I right?

Reply:
If I'm wrong, I would appreciate if you could help me on how to fix this one.

Reply:
Indeed changes to `BackoffOptionsImpl` should be fine, so you can exclude those from MiMa as described in the CONTRIBUTING.md. It looks like you have some similar problems in `Backoff`, though, which do need to be addressed. Does that help?

Reply:
Yes, it does. Thanks alot.